sui/its.js

```diff
@@ -811,6 +831,49 @@ async function checkVersionControl(keypair, client, config, contracts, args, opt
     }
 }
 
+async function mintCoins(keypair, client, config, contracts, args, options) {
+    const [coinPackageId, coinPackageName, coinModName, amount, receiver] = args;
+
+    const walletAddress = keypair.toSuiAddress();
+
+    const coinType = `${coinPackageId}::${coinPackageName}::${coinModName}`;
+
+    await checkIfCoinExists(client, coinPackageId, coinType);
+
+    const { data } = await client.getOwnedObjects({
+        owner: walletAddress,
+        filter: { StructType: `0x2::coin::TreasuryCap<${coinType}>` },
+        options: { showType: true },
+    });
+
+    if (!Array.isArray(data) || data.length === 0) {
+        throw new Error('TreasuryCap object not found for the specified coin type.');
+    }
+
+    const treasury = data[0].data?.objectId ?? data[0].objectId;
+
+    const txBuilder = new TxBuilder(client);
+    await txBuilder.moveCall({
+        target: `${coinPackageId}::${coinPackageName}::mint`,
+        arguments: [treasury, amount, receiver],
+    });
+
+    const response = await broadcastFromTxBuilder(txBuilder, keypair, `Mint ${coinPackageId}`, options);
+
+    const balance = await client.getBalance({
+        owner: walletAddress,
+        coinType: `${coinPackageId}::${coinPackageName}::${coinModName}`,
+    });
+
+    console.log(`💰 my token balance ${balance.totalBalance}`);
+
+    const coinChanged = response.objectChanges.find((c) => c.type === 'created');
+
+    console.log('New coin object id:', coinChanged.objectId);
```

Comment:
Let's update these all console logs to use the `printInfo` function to keep the scripts consistent